### DISCUSSION TITLE
How to overwrite the CMD when using a service container, since there is no `service.<service-name>.command` key available?

### DISCUSSION THREAD

**Discussion by LeoNero** [community/community#31825] [Actions]
Title

**Comment by airtower-luna** [comment]
You should be able to override the entrypoint using [`jobs.<job_id>.services.<service_id>.options`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idservicesservice_idoptions). And you can always start the container in a step (`docker run`, etc.), just make sure to attach it to the job network if you have any other containers.

**Comment by gmile** [comment]
Slightly more discussion on the topic is happening here: https://github.com/orgs/community/discussions/26688
